### DISCUSSION TITLE
Dynamic Products For v4.8.x

### DISCUSSION THREAD

**Discussion by Blair2004** [Blair2004/NexoPOS#820] [Polls]
We're would like to add the possibility to turn the quick product into a dynamic product. A dynamic product will have the price dynamically set using a rate (percentage) defined.

![image](https://user-images.githubusercontent.com/5265663/172011289-4667d8b7-def5-44ce-acf7-54e20029473d.png)

While adding to the cart, that product will price will be computed using the subtotal % rate, add added to the subtotal. 

https://user-images.githubusercontent.com/5265663/172016464-d1872e61-7356-4d60-a43d-0e667119431c.mp4

We have some possible scenarios where this can be used:

- Service Charge ( a percentage of the subtotal)
- Packaging service (for which the price depends on a percentage)
- Untracked Commission (there is already a commission module)

We're wondering if that's a feature that might be useful for your business. Please vote down below.

**Comment by nzian** [comment]
My case is Delivery charge is dynamic and its based on delivery area and subtotal. For example: 
area: xyz  subtotal >= $25 delivery charge free otherwise delivery charge $2.
area: mnq  delivery charge $10 and subtotal >= $100 delivery charge $8, subtotal >$200 delivery charge $6
Does this cases cover with dynamic products?
